schema/properties/best.json

```diff
@@ -1,10 +1,10 @@
 {
-    "schemaId": "properties/best-property",
+    "schemaId": "properties/best",
```

Comment:
If the follow the `in-memory-entity/base` convention above, the name of the folder should be "property", not "properties"

Reply:
done